spec/models/hyrax/permission_template_access_spec.rb

```diff
@@ -54,4 +54,106 @@
       end
     end
   end
+
+  context 'with a user that is an admin set viwer' do
+    let(:agent_type) { 'user' }
+    let(:agent_id) { 'foobar' }
+    let(:access) { 'view' }
```

Comment:
Use Hyrax::PermissionTemplateAccess::VIEW instead of 'view'.  I know that other tests just use the string, but whenever we touch a test or add more, we should switch to using the constant.